packages/backend-modules/access/lib/grants.js

```diff
@@ -85,6 +85,35 @@ const grantPerks = async (grant, recipient, campaign, t, pgdb, redis, mail) =>
     { concurrency: 1 },
   )
 
+const revokePerks = async (grant, recipient, campaign, pgdb) =>
```

Comment:
```suggestion
const revokePerks = (grant, recipient, campaign, pgdb) =>
```

`async` keywords is not necessary, since a Promise is returned and you're not awaiting anything in _that_ function.

(Can also go for `grantPerks` and lies in my inaptiude to write clean code.)